fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/CalculateIncomeAndExpenseBookingIntegrationTest.java

```diff
@@ -0,0 +1,161 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.fineract.integrationtests;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.fineract.accounting.closure.command.GLClosureCommand;
+import org.apache.fineract.accounting.closure.data.IncomeAndExpenseBookingData;
+import org.apache.fineract.accounting.closure.data.IncomeAndExpenseJournalEntryData;
+import org.apache.fineract.accounting.closure.exception.RunningBalanceNotCalculatedException;
+import org.apache.fineract.accounting.closure.service.CalculateIncomeAndExpenseBookingImpl;
+import org.apache.fineract.accounting.closure.service.IncomeAndExpenseReadPlatformService;
+import org.apache.fineract.infrastructure.core.api.JsonCommand;
+import org.apache.fineract.organisation.office.service.OfficeReadPlatformService;
+import org.joda.time.LocalDate;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+
+
+public class CalculateIncomeAndExpenseBookingIntegrationTest {
```

Comment:
Isn't this actually a Unit and not an Integration Test? It doesn't seem to make any REST calls (as far as I can tell). If that is correct, then how about moving this to `src/test` instead of `src/integregationTest`?